og.module

```diff
@@ -150,9 +150,8 @@ function og_entity_create_access(AccountInterface $account, array $context, $bun
     }
 
     $handler = Og::getSelectionHandler($field_definition);
-
-    if ($handler->getReferenceableEntities()) {
-      return AccessResult::neutral();
+    if ($handler->countReferenceableEntities()) {
+      return AccessResult::allowed();
```

Comment:
Two ideas for simplification, that we need to think about:

1. Allow having only a single audience field (back to D6 days 😄 )
1. Once a bundle has an audience field, that field is required. So you can't for example have a post outside of a group context.

(1) Is removing some advanced features, which I have honestly never seen used, like creating different memberships based on the field group was selected from. Or allowing to add a group content to groups of different entity types.
(2) Which is less important _might_ be used for example for adding articles on groups and articles for the site-wide. I can think of easy workflows to keep that (e.g. create a "site-wide" group)

/cc @pfrenssen @RoySegall